templates/README.md.tpl

```diff
@@ -0,0 +1,50 @@
+### Repos
+
+{{- range rss "https://github.com/charmbracelet/bubbles/releases.atom" 1 }}
+- Bubbles: TUI components for Bubble Tea 🍡 [{{ .Title }}]({{ .URL }}) (_released {{ humanize .PublishedAt }}_)
+{{- end }}
+{{- range rss "https://github.com/charmbracelet/bubbletea/releases.atom" 1 }}
+- Bubble Tea: A powerful little TUI framework 🏗 [{{ .Title }}]({{ .URL }}) (_released {{ humanize .PublishedAt }}_)
+{{- end }}
+{{- range rss "https://github.com/charmbracelet/charm/releases.atom" 1 }}
+- Charm: The Charm Tool and Library 🌟 [{{ .Title }}]({{ .URL }}) (_released {{ humanize .PublishedAt }}_)
```

Comment:
I feel like these descriptions should be more descriptive of what the app or library does. I think it would be great if people could look at this README and get ideas on what they could do with the libs without having to dig for more info.